### PR TITLE
Section model: fix a return value

### PR DESCRIPTION
**Pull request by breville**
Fix return value of add_and_remove_student, as suggested in code review feedback for https://github.com/code-dot-org/code-dot-org/pull/16586

### CODE REVIEW

File: dashboard/app/models/sections/section.rb [hunk 1]
```diff
@@ -178,6 +178,7 @@ def add_and_remove_student(student, old_section)
 
     old_follower.destroy
     add_student student
+    true
```
Comment (ashercodeorg) [inline dashboard/app/models/sections/section.rb]: It seems like this needs to be slightly more complex, e.g.:
```
add_student_success = add_student student
add_student_success == 'ADD_STUDENT_SUCCESS'
```

Comment (ashercodeorg) [reply dashboard/app/models/sections/section.rb]: Otherwise, you are returning true even if the add student was unsuccessful.

**Comment by breville** [comment]
fwiw, I don't think we use this function any more...

**Comment by islemaster** [comment]
Closing in favor of https://github.com/code-dot-org/code-dot-org/pull/27491

**Pull request closed without merging**